packages/tools/devtools/devtools-browser-extension/src/devtools/TelemetryLogging.ts

```diff
@@ -75,11 +76,17 @@ export class OneDSLogger implements ITelemetryBaseLogger {
 			httpXHROverride: fetchHttpXHROverride,
 		};
 
+		const instrumentationKey = (process.env.DEVTOOL_TELEMETRY_TOKEN as string) ?? "";
```

Comment:
Let's add a comment here that explains what happens here in the end; how `process.env` here is not an issue because dotenv-webpack will replace this at bundle time, so we're not really using `process` in a browser environment.

Reply:
Got it! Adjusting accordingly.